pipeline/src/client.py

```diff
@@ -114,9 +120,17 @@ def main(self, read_env_data=False):
     #######################################################
 
     def create_hive(self):
-        self.flags.create_table()
-        self.service_periods.create_table()
-        self.flagged.create_table()
+        if (self._output_type == "aperture" or self._output_type == "both"):
+            self.flags.create_table()
+            self.service_periods.create_table()
+            self.flagged.create_table()
+
+        #When dealing with csv and hive creation, only data available right away is flags, so we only save flags to csv when "creating" csv-hive
+        if (self._output_type == "csv" or self._output_type == "both"):
+            if not self.flags.write_csv(self._output_path):
+                print("Error saving Flags to csv.")
+
+            self.service_periods.create_table()
```

Comment:
Why do you create the service_periods table here and not the other two?

Reply:
I actually forgot to remove creating service periods from here. I'm not sure what's the desired behavior. DO we want to create 2 empty tables in this case?

Reply:
In the case of service_periods, I believe that we query and insert into it directly as we are processing data, so it needs to exist. I guess the other two tables don't need to be explicitly created, but maybe for the sake of consistency they should be created anyways. I'm thinking maybe don't even provide them the option of csv only and always require them to do either database only or both. 

Unless you feel like changing service_periods so that it stores info in memory instead and only write when told to explicitly, in which case I think that's a pretty reasonable thing to do.

Reply:
The other 2 tables are created, but not from hive. When we run process_data, it creates both flagged and service_periods tables (csv's). I just didn't see a point in creating an empty csv, thought it can be easily fixed if there is such a need.

Reply:
Probably better to explicitly create them instead of relying on secondary processes to do so.